src/Web/WebClient.php

```diff
@@ -53,81 +53,108 @@ class WebClient
 	const ANDROIDTABLET = 22;
 	const EDGE          = 23;
 	const BLINK         = 24;
+	const EDG           = 75;
```

Comment:
75?

Also, is there a better name that can be used aside from "EDG"?

Reply:
this is the version agent: Edg/75.0.107.0

full agent: `Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/75.0.3738.0 Safari/537.36 Edg/75.0.107.0`

Reply:
Oh, well, OK then on the name.

The numbers don't really correspond to first versions or anything, it's just sequential.  So the EDG constant doesn't need to be 75, it can be 25 and not really matter (I wish strings had been used instead of less relevant numbers, but changing constant values is a B/C break so that ship's sailed).

Reply:
The Edge version is now at 44, The chromium - alpha version will be shipped with 75.

I have updated the file.